airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -155,10 +155,11 @@ class ClusterGenerator:
     """
 
     # pylint: disable=too-many-arguments,too-many-locals
+    @DataprocHook.fallback_to_default_project_id
```

Comment:
Is it works for operators?

Reply:
No, it doesn't, so do we want to add such logic? This will require to initialize the hook somewhere so we can access the project_id. I personally don't see a lot of value in it

Reply:
Or probably we should refactor the operator, so there's no need to provide the project in two places: as argument and in cluster definition